app/controllers/users_controller.rb

```diff
@@ -109,8 +109,12 @@ def confirm_destroy
     if destroy_token.blank?
       flash[:settings_notice] = "Your token has expired, please request a new one. Tokens only last for 12 hours after account deletion is initiated."
       redirect_to user_settings_path("account")
-    else
-      raise ActionController::RoutingError, "Not Found" unless destroy_token == params[:token]
+    elsif destroy_token != params[:token]
+      Honeycomb.add_field("user_id", @user.id)
```

Comment:
We don't actually need this since [we already log the `current_user_id` when we authenticate the user](https://github.com/forem/forem/blob/master/app/controllers/application_controller.rb#L73).
```ruby

def authenticate_user!
  if current_user
    Honeycomb.add_field("current_user_id", current_user.id)
    return
  end 

  respond_to do |format|
    format.html { redirect_to sign_up_path }
    format.json { render json: { error: "Please sign in" }, status: :unauthorized }
  end
end
```

Reply:
TIL - thank you! I've made this change and updated the spec accordingly, too. 😊